src/java.desktop/unix/classes/sun/awt/X11/XWM.java

```diff
@@ -1369,6 +1369,9 @@ Insets guessInsets(XDecoratedPeer window) {
               case UNITY_COMPIZ_WM:
                   res = new Insets(28, 1, 1, 1);
                   break;
+              case MUTTER_WM:
+                  res = new Insets(37, 0, 0, 0);
+                  break;
               case MOTIF_WM:
               case OPENLOOK_WM:
```

Comment:
Out of scope for this fix but in a future fix I think we can get rid of at least
(1) LG3D
(2) OPENLOOK
and very probably
(3) MOTIF
and perhaps also
(4) CDE

They are all obsolete.